openedx_lti_tool_plugin/deep_linking/views.py

```diff
@@ -122,14 +125,21 @@ def get(
 
         """
         try:
+            # Get message from cache.
             message = self.get_message_from_cache(request, launch_id)
+            # Validate message.
             validate_deep_linking_message(message)
-
+            # Get identity claims from launch data.
+            iss, aud, _sub, _pii = get_identity_claims(message.get_launch_data())
+            # Render form template.
```

Comment:
@kuipumu Add blank line.

Reply:
@anfbermudezme I don't think is necessary to add a blank line